exampleCourse/questions/demo/insertEmbeddedVideo/question.html

```diff
@@ -0,0 +1,19 @@
+<pl-question-panel>
+  <p>This is an awesome video with an introduction to PrairieLearn.
+     Learn about how to create randomized assessments, visualize your course statistics and more!</p>
```

Comment:
```suggestion
     Learn about how to create randomized assessments, visualize your course statistics, and more!</p>
```

I like using serial comma :slightly_smiling_face: